### DISCUSSION TITLE
Simple thousand separator

### DISCUSSION THREAD

**Discussion by Thorgal01** [beholdr/maska#96] [Q&A] [answered]
Hello,

I have the version 1.5.1, and I wanted to have a simple mask for thousand separator with space, like 800 000.
I want to have only numbers without any decimals, how I have to write the mask?

Thanks in advance for your help.
Best regards

**Comment by beholdr** [comment, marked as answer]
Why don’t you want to use v2? It has reverse mode that allows to make such masks very simple:

``` html
<input
  v-maska
  data-maska="9 99#"
  data-maska-tokens="9:[0-9]:repeated"
  data-maska-reversed
>
```

**Reply by Thorgal01** [reply]
Thanks for the response. 
I tried with the v2, but I was not able to make it working with vuetify in a v-text-field, I always had an error regarding "eager".... 
That's why I rollback to v1.

**Reply by beholdr** [reply]
I don’t think it can be done with v1.
Could you create reproducible example in sandbox with v2 and vuetify?

**Reply by Thorgal01** [reply]
Yes, you can test from this link: [https://codesandbox.io/s/kind-goldwasser-3rqpm2?file=/src/components/TestMask.vue]

When you change the value of the textbox you have the error: "Cannot read properties of undefined (reading 'eager')"
And I have the same error in my local project.

If I use a "normal" input it works.

**Reply by beholdr** [reply]
I've fixed Vuetify error on v2.0.1 but it still could be buggy because of v-model and Vuetify events.

**Reply by beholdr** [reply]
@Thorgal01 could you please try v2.1 with vuetify? It should work with Vuetify.

**Reply by Thorgal01** [reply]
Yes now it works fine. Thanks for your support

**Reply by rspeicher** [reply]
Can I suggest adding this as a basic example to the demo site? This felt to me like something that was a common use case for Maska and I was surprised how long it took me to find this discussion.

**Reply by DJafari** [reply]
@beholdr if we want max two optional decimal and thousand separator, can you give a help for that ?

example : 

`12345.3` => `12,345.3`
`12345` => `12,345`
`123456.32` => `123,456.32`

note : number length is uknown
